### DISCUSSION TITLE
How to move sent/received messages from Message+ to Fossify messenger.

### DISCUSSION THREAD

**Discussion by Sin-Shadow-Fox** [FossifyOrg/General-Discussion#244] [Q&A]
How do i transfer my sent and received messages from Message+ to Fossify as Message+ is shutting down on November 7th. I don’t want to lose several years of messages so i need a solution before the 7th. Please help.

**Comment by Aga-C** [comment]
Please raise this question in [Messages](https://github.com/FossifyOrg/Messages/discussions).
